projects/storefrontlib/src/cms-components/navigation/navigation/navigation.service.ts

```diff
@@ -162,11 +162,15 @@ export class NavigationService {
 
   /**
    *
-   * Gets the URL or link to a related item (category)
+   * Gets the URL or link to a related item (category),
+   * also taking into account content pages (contentPageLabelOrId)
+   * and product pages (productCode)
    */
   private getLink(item): string | string[] {
```

Comment:
It might be good time to make this method protected. And to add tests for new cases